test/Conversion/VectorToAIEVec/test_redundant_store.mlir

```diff
@@ -0,0 +1,48 @@
+// RUN: aie-opt %s -affine-super-vectorize="virtual-vector-size=16" --convert-vector-to-aievec="aie-target=aieml" -lower-affine -canonicalize | FileCheck %s
```

Comment:
We don't need to test the optflow optimization, it's already being tested in LLVM (mlir/test/Dialect/Vector/vector-transferop-opt.mlir), together with more complex cases.

Reply:
Not sure what you mean here. How do you want me to test this test case.